### PR TITLE
refactor: docs design

### PR DESCRIPTION
**Pull request by iCrawl**
**Please describe the changes this PR makes and why it should be merged:**

**Status and versioning classification:**

### CODE REVIEW

File: packages/website/package.json [hunk 1]
```diff
@@ -47,6 +47,13 @@
 	},
 	"homepage": "https://discord.js.org",
 	"dependencies": {
+		"@emotion/react": "^11.10.0",
+		"@emotion/server": "^11.10.0",
+		"@mantine/core": "^5.1.5",
+		"@mantine/hooks": "^5.1.5",
+		"@mantine/next": "^5.1.5",
```
Comment (imranbarbhuiya) [inline packages/website/package.json:54]: Have you tried https://www.npmjs.com/package/nextjs-progressbar, it doesn't need any extra configurations. You just need to add `<NextNProgress />` in `_app.ts`.

Comment (imranbarbhuiya) [reply packages/website/package.json:54]: I selected the wrong line number. This suggestion was for `@mantine/nprogress`

Comment (iCrawl) [reply packages/website/package.json:54]: I'd rather stay in the same UI framework.

File: packages/website/src/components/SidebarItems.tsx [hunk 2]
```diff
@@ -81,42 +65,54 @@ function resolveIcon(item: keyof GroupedMembers) {
...
-										>
-											{member.name}
-										</a>
-									</Link>
-									<div>
+				.map((group, idx) => (
+					<Section key={idx} title={group} icon={resolveIcon(group)}>
+						{groupItems[group].map((member, i) => (
+							<Link key={i} href={member.path} passHref>
+								<UnstyledButton className={classes.link} component="a" onClick={() => setOpened((o) => !o)}>
```
Comment (imranbarbhuiya) [inline packages/website/src/components/SidebarItems.tsx:104]: different color for the selected member is removed. Is it intentional?

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

File: packages/website/src/components/SidebarItems.tsx (hunk 2 above)
Comment (iCrawl) [reply packages/website/src/components/SidebarItems.tsx:104]: For now, yes.

**Pull request merged**